src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,73 @@
-export const Pagination = () => {};
+type PaginationType = {
+  total?: number;
+  perPage?: number;
+  currentPage?: number;
+  pages: string[][];
```

Comment:
The `pages` prop should not be used in the `Pagination` component. Instead, use `total` and `perPage` to calculate the number of pages.